src/autoscaler/db/db.go

```diff
@@ -34,3 +34,8 @@ type HistoryDB interface {
 	CanScaleApp(appId string) (bool, error)
 	Close() error
 }
+
+type ScheduleDB interface {
```

Comment:
when the active schedule end is handled properly, the record will be deleted.  This will be addressed together with user story #131776923